src/phel/json.phel

```diff
@@ -1,33 +1,45 @@
 (ns phel\json
-  (:use \Exception))
+  (:use \RuntimeException))
 
-(defn valid-key? [v]
+(defn valid-key?
+  "Check if v is a valid json key."
+  [v]
   (or (int? v) (float? v) (symbol? v) (keyword? v) (string? v)))
 
+(declare encode-value [x])
+
+(defn- encode-value-iterable [x]
+  (let [arr (php/array)]
+    (foreach [k v x]
+      (when-not (valid-key? k)
+        (throw (php/new RuntimeException "Key can only be an integer, float, symbol, keyword or a string.")))
+      (php/aset arr (encode-value k) (encode-value v)))
+    arr))
+
+(defn- encode-value-with-name [x]
+  (str (php/-> x (getName))))
```

Comment:
We now have the `name` function. This can be used instead of `getName` (https://phel-lang.org/documentation/api/#name)

Reply:
I forgot about that, thank you! I will use that instead of this new private function `encode-value-with-name`.